reana-ui/src/pages/workflowDetails/WorkflowDetails.js

```diff
@@ -103,9 +103,17 @@ function WorkflowDetails() {
 
   const panes = [
     {
-      menuItem: { key: "logs", icon: "terminal", content: "Logs" },
+      menuItem: { key: "job-logs", icon: "terminal", content: "Job Logs" },
       render: () => <WorkflowLogs id={workflow.id} />,
     },
+    {
+      menuItem: {
+        key: "engine-logs",
+        icon: "cogs",
+        content: "Engine Logs",
```

Comment:
What about "Engine logs" case in the visible menu title,  to be consistent with BrE style rather than AmE style of title headings.